### PR TITLE
Update DOCKER_DASH_COMPOSE_VERSION default to v2

### PR DESCRIPTION
**Pull request by leobaldoneto**
Changed it because devcontainer-feature.json documentation indicates that it's the default version.

### CODE REVIEW

File: src/docker-outside-of-docker/install.sh [hunk 1]
```diff
@@ -10,7 +10,7 @@
 DOCKER_VERSION="${VERSION:-"latest"}"
 USE_MOBY="${MOBY:-"true"}"
 MOBY_BUILDX_VERSION="${MOBYBUILDXVERSION:-"latest"}"
-DOCKER_DASH_COMPOSE_VERSION="${DOCKERDASHCOMPOSEVERSION:-"v1"}" # v1 or v2 or none
+DOCKER_DASH_COMPOSE_VERSION="${DOCKERDASHCOMPOSEVERSION:-"v2"}" # v1 or v2 or none
```
Comment (samruddhikhandale) [inline src/docker-outside-of-docker/install.sh:13]: Can you help bump the patch version for this Feature? Thanks! 

https://github.com/devcontainers/features/blob/a01cd8e70440dc72e96c72eaa07ab36a9e291b3a/src/docker-outside-of-docker/devcontainer-feature.json#L3

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]
Thanks for the update, left one minor comment.

**Comment by leobaldoneto** [comment]
@microsoft-github-policy-service agree

**Review by samruddhikhandale — APPROVED** [review]
Looks good! thanks

**Pull request merged**